traffic_portal/app/src/common/modules/table/certExpirations/table.certExpirations.tpl.html

```diff
@@ -0,0 +1,70 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div class="x_panel">
+    <div class="x_title">
+        <ol class="breadcrumb pull-left">
+            <li class="active">Certificate Expirations</li>
+        </ol>
+        <div class="pull-right">
+            <div class="form-inline" role="search">
+                <input id="quickSearch" name="quickSearch" type="search" class="form-control text-input" placeholder="Quick search..." ng-model="quickSearch" ng-change="onQuickSearchChanged()" aria-label="Search"/>
+                <div class="input-group text-input">
+                    <span class="input-group-addon">
+                        <label for="days">Days</label>
+                    </span>
+                    <input id="days" name="days" type="number" min="1" class="form-control" placeholder="" ng-model="days" ng-change="onDaysChanged(days)" aria-label="Days"/>
```

Comment:
You don't need `aria-label` since the element is explicitly labeled. An empty `placeholder` is the default, so it need not be specified.

Reply:
Passing `days` to the `onDaysChanged` expands expressions of non-constant variables to scope properties; this is calling `$scope.onDaysChanged($scope.days)`. This is a bit of a nitpick, but since `$scope.days` is in-scope of the function `$scope.onDaysChanged`, you don't need to pass it an argument, you can just access the model value directly.

Reply:
i removed the `days` aspect completely and used the common ag grid instead